modules/ROOT/partials/release-notes/6.0-release-notes-summary.adoc

```diff
@@ -276,7 +276,7 @@ The following elements were previously deprecated and have, with this release, b
 
 | `validate`                            | Schema option |
 
-| `XHR`                                 | API           | Any remaining `XHR` users have been replaced with `fetch`.
+| `XHR`                                 | API           | Any remaining `XHR` uses have been replaced with https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API[`fetch`].
```

Comment:
Should it just be `usages` ?

Reply:
*use*: the act of using.

*uses*: 3rd-person singular simple present indicative of *use*.

*usage*: 1. A custom or established practice; 2. the act of using.

*usages*: plural of *usage*.

*Use* and *usage* are near-perfect synonyms.

I’ll weakly argue for *uses* in this case only because a number of dictionaries present ‘custom or established practice’ as the more common usage (see what I did there).